src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -8,7 +8,58 @@ import { metricError } from "config/metric-names";
 import { AppInstallation, FailedAppInstallation } from "config/interfaces";
 import { createAppClient } from "~/src/util/get-github-client-config";
 import { booleanFlag, BooleanFlags } from "config/feature-flags";
-import { isGitHubCloudApp } from "~/src/util/jira-utils";
+import { GitHubServerApp } from "models/github-server-app";
+
+interface FailedConnection {
```

Comment:
I know these were here before but just for future reference: if you want to move something because it makes more sense for the 'flow' that's fine, but please do it in a separate PR. It makes it very hard to tell what is original code and what is new code when everything has been moved around.